rpc-test/tests/rpc.rs

```diff
@@ -281,12 +282,20 @@ fn test_rpc_subscriptions() {
     let rt = Runtime::new().unwrap();
     let rpc_pubsub_url = test_validator.rpc_pubsub_url();
     let signature_set_clone = signature_set.clone();
+    let account_set_clone = account_set.clone();
+    let signature_subscription_ready = Arc::new(AtomicUsize::new(0));
+    let account_subscription_ready = Arc::new(AtomicUsize::new(0));
+    let signature_subscription_ready_clone = signature_subscription_ready.clone();
+    let account_subscription_ready_clone = account_subscription_ready.clone();
+
     rt.spawn(async move {
         let pubsub_client = Arc::new(PubsubClient::new(&rpc_pubsub_url).await.unwrap());
 
         // Subscribe to signature notifications
         for signature in signature_set_clone {
             let status_sender = status_sender.clone();
+            let signature_subscription_ready_clone_clone =
+                signature_subscription_ready_clone.clone();
```

Comment:
I don't think there's an alternative to the multi-clone, but you could do this to make var names less intense:
```suggestion
            let signature_subscription_ready_clone =
                signature_subscription_ready_clone.clone();
```

Reply:
ah! yeah. I can just do name shadowing. Thank you for pointing it out. fixed at [7585b7f](https://github.com/solana-labs/solana/pull/27394/commits/7585b7f40800cd1a85d23f6a10070d0eb086842b)